tests/pytorch/test_loss.py

```diff
@@ -78,8 +78,8 @@ def test_register_loss_functional(out_dir):
     loss_tensor = trial.tensor("nll_loss_output_0")
 
     # Capture ['nll_loss_output_0']
-    assert len(trial.tensor_names()) == 2
-    assert len(loss_coll.tensor_names) == 2
+    assert len(trial.tensor_names()) >= 1
+    assert len(loss_coll.tensor_names) >= 1
```

Comment:
Can you add a comment here with a reference to the issue?

Reply:
done in the latest commit